modules/ROOT/pages/managing-permissions.adoc

```diff
@@ -8,49 +8,49 @@ This section explains how permissions work across different products and APIs ma
 
 . *Product Permissions*
-* Default permissions for each Anypoint Platform product (Runtime Manager, Data Gateway, etc). They are *environment specific* – they grant you the ability to do something within a particular environment, but not to the entire organization. +
-Product permissions can be assigned to custom roles, or to CloudHub roles.
+* Default permissions for each Anypoint Platform product (Runtime Manager, Data Gateway, and so on). They are _environment specific_ and grant you the ability to do something within a particular environment, but not to the entire organization. +
+Product permissions can be assigned to teams, custom roles, or to CloudHub roles.
 +
-[NOTE]
---
-Access to Exchange is handled through the *Exchange Administrators*, *Exchange Contributors*, and *Exchange Viewers* xref:roles.adoc[roles]. It is not possible to grant individual Exchange permissions to other roles. Consult the xref:exchange::to-set-permissions.adoc[product documentation] to learn more about this.
---
 . *API Permissions*
 +
 API Permissions vary depending on the API Manager version you are using.
 +
 .. *Permissions in API Manager 1.x*
 +
-* Default permissions for each API managed from  Anypoint Platform. They can be specific for an API version, or they can be extended to all versions. +
-You can manage user access based on a particular API version, but you cannot extend those permissions to the entire organization. To grant user permissions to all APIs, add them to the *API Versions Owner* and *Portals Owner* roles.
-* You can assign user permissions to edit or view individual API versions or API portals using the *API Version Owner* and the *Portal Viewer* permission.
+* Default permissions for each API managed from Anypoint Platform. They can be specific for an API version, or they can be extended to all versions. +
+You can manage user access based on a particular API version, but you cannot extend those permissions to the entire organization. To grant user permissions to all APIs, add them to teams that have the the API Versions Owner and Portals Owner permissions.
+* You can assign user permissions to edit or view individual API versions or API portals using the API Version Owner and the Portal Viewer permissions.
 +
 [NOTE]
 --
-These two API permissions grant the same scopes as the *API Versions Owner* and *Portals Viewer* roles, but limited to the API and version that you defined. +
-Learn about these specific scopes in the xref:roles.adoc#permission-scopes-for-default-roles[roles article].
+These two API permissions grant the same scopes as the API Versions Owner and Portals Viewer roles, but limited to the API and version that you defined. +
+For more information about these specific scopes, see xref:roles.adoc#permission-scopes-for-default-roles[Roles].
```

Comment:
I'd like to see some definition or clarity around permission "scopes"...maybe just renaming to "permission scopes"?